libs/design/guides/getting-started.md

```diff
@@ -49,28 +49,26 @@ There is a minimal required global style for the Daffodil Design System to opera
 > For more information on our approach to these kinds of styles, see the [Global Styles guide.](/libs/design/guides/foundations/global-styles.md)
 
 ## Add a theme
-A theme must be configured in order for the components to work properly. The components in the design library can be configured with customized colors in addition to a dark and light mode for those same colors.
+A theme must be configured in order for Daffodil Design components to work properly.
```

Comment:
`Daffodil Design` sounds awkward, maybe just always reference the package name? `@daffodil/design`?

Reply:
What's the difference between Daffodil Design and `@daffodil/design`? Reading it, it sounds the same. I don't think the design system should always be referred to via the package name. That's a very developer thing, and the design system is not exclusively catered to developers.